### ISSUE TITLE
Раздел Практические вопросы

### ISSUE THREAD

**Issue by matzuk** [AndroidArchitecture/AndroidArchitectureBook#15]
На данный момент раздел представляет собой сборник вопросов-ответов с соответствующим хэштэгами.  В идеале, как мне видется, это должна быть статья с более глубоким погружением в отличии от раздела с Теорией. То есть статья должна собой представлять подробное рассмотрение каждого слоя с его особенностями, рассмотрение того, как ложатся андроидовские компоненты и т.д.

_Сейчас принимаются идеи, концепции и люди, кто хочет пилить, дабы в дальнейшем не делать двойную работу._

**Comment by ardmn** [comment]
Всем привет . Скажите пожалуйста почему в примере с авторизацией thread pool имеет размер в 3 потока? Как понять сколько потоков лучше выбрать для  thread pool и почему ?

**Comment by AlexeyKorshun** [comment]
Доброго дня, кто как обрабатывает смерть процесса в модели?

**Comment by matzuk** [comment]
@ardmn это чисто для примера я я взял три потока.
По поводу размера thread pool - все довольно индивидуально, зависит от задачи. За основу расчета можно взять пример AsyncTask. Посмотрите, как там считают. Ну или же используйте готовые фабричные методы.

**Comment by matzuk** [comment]
@AlexeyKorshun а какая задача? Если нужно, чтобы данные переживали смерть процесса, пишите в БД (реализация под Репозиторием), да и все. Или что-то еще есть?

**Comment by AlexeyKorshun** [comment]
@matzuk да, но не совсем эти данные логично писать в репозиторий, savedInstanceState куда лучше подхоит для этого, но прокидывать эти колбеки в модель тоже как то сомнительно.
